### ISSUE TITLE
[L4] Parameter-less rules getting parameters set

### ISSUE THREAD

**Issue by patrickcarlohickman** [formers/former#288]
While debugging another issue, when I dumped out the Former rules array I noticed that some of the rules were getting parameters that didn't make sense.  For example, a 'required' rule with parameters.  This is probably a harmless bug, but was a bit confusing when I saw it.

I believe that the $parameters variable should be set to an empty array at the beginning of each loop iteration in the withRules() method.  With the current code, all the parameter-less rules will get assigned the last set of parameters that were processed.

former/src/Former/Former.php
```php
public function withRules()
{
  // snip

  foreach ($expFieldRules as $rule) {
    $parameters = array();

    // snip

    $this->rules[$name][$rule] = $parameters;
  }

  // snip
}
```

**Comment by claar** [comment]
Closing old inactive issues to make project maintainable.

**Comment by patrickcarlohickman** [comment]
Just for reference, this issue was fixed in commit 2fb11a71d69ca53b93f9468c4a9beb59f97f4902, which closed duplicate issue #480. Thanks.
